packages/smart/contracts/turbo/AMMFactory.sol

```diff
@@ -6,15 +6,26 @@ import "../balancer/BFactory.sol";
 import "../libraries/SafeMathUint256.sol";
 import "./AbstractMarketFactory.sol";
 import "../balancer/BNum.sol";
+import "../rewards/MasterChef.sol";
 
-contract AMMFactory is BNum {
+contract AMMFactory is BNum, Ownable {
     using SafeMathUint256 for uint256;
 
     uint256 private constant MAX_UINT = 2**256 - 1;
+    uint256 private constant MIN_INITIAL_LIQUIDITY = BONE * 100;
+
+    mapping(address => bool) private approvedMarketFactories;
+
+    // Using constant so all pools get equal portion of rewards.
+    uint256 private constant _allocPoint = 5;
```

Comment:
Why 5?

Reply:
At this point it is a random number that I used. Filed a ticket to follow up: https://github.com/AugurProject/turbo/issues/742